README.md

```diff
@@ -15,3 +13,309 @@ To begin, clone this repository. `cd` into your chosen folder, and run the follo
+
+Parameter Name | Parameter Type | Description
+--- | --- | ---
+selector | String | The CSS/Xpath selector to locate the element.
+message | String | _optional_ The message to output.
+callback | Function | _optional_ A function to call after the current command finishes execution.
+
+```
+this.demoTest = function (browser) {
+    browser.assert.elementsPresent('#x-root', '#x-header', '#x-footer');
+};
+```
+
+#### elementsVisible(selector, message, callback)
+
+The `elementsVisible` assertion checks if one or more selectors are visible. It returns a list of one or more selectors that are not visible on the page.
+
+Parameter Name | Parameter Type | Description
+--- | --- | ---
+selectors | String | The CSS/Xpath selector to locate the element.
+callback | Function | _optional_ A function to call after the current command finishes execution. 
+
+```
+this.demoTest = function (browser) {
+    browser.assert.elementsVisible('#x-root', '#x-head');
+};
+```
+
+#### templateName(expected, message, callback)
+
+The `templateName` assertion checks if the given template name is correct.
+
+Parameter Name | Parameter Type | Description
+--- | --- | ---
+expected | String | The expected value of the attribute to check.
+message | String | _optional_ A log message to display in the output. If the parameter is not specified, a default message is displayed.
+callback | Function | _optional_ A function to call after the current command finishes execution. 
+
+```
+this.demoTest = function (client) {
+    browser.assert.templateName('home');
+};
+```
+
+## Commands
+
+#### get(url, callback)
+
+The `get` command combines the `url` and `waitUntilMobified` functions. This command uses the `browser.execute` function to run code within the client browser. It then waits for the `Mobify` object to be present on the page
+
+Parameter Name | Parameter Type | Description
+--- | --- | ---
+url | String | The URL to load
+callback | Function | _optional_ A function to call after the current command finishes execution. 
+
+```
+this.demoTest = function (browser) {
+    browser.get('http://www.test.com');
+};
+```
+
+#### getMobifyEvaluatedData(callback)
+
+The `getMobifyEvaluatedData` command uses the `waitForCondition` method to retrieve the `Mobify.evaluatedData` from the client browser.
+
+Input 
+
+Parameter Name | Parameter Type | Description
+--- | --- | ---
+callback | Function | _optional_ A function to call after the current command finishes execution. 
+
+Output
+
+Parameter Name | Description
+--- | --- | ---
+Object | Returns the client object after `waitUntilMobified` executes on it with the specified parameters.
+
+```
+this.demoTest = function (browser) {
+    browser.getMobifyEvaluatedData();
+};
+```
+
+#### htmlCapture(message, callback)
+
+The `htmlCapture` command saves desktop HTML to `tests/system/integration/fixtures`, for use in integration tests. This command should be run using an environment with a desktop user agent. 
```

Comment:
The `htmlCapture` is a helper command that saves a pages HTML to `tests/system/integration/fixtures`.
This command was designed to collect and save HTML files for use in integration tests. The initial design of his command was to be run using a nightwatch environment with a desktop user agent.

However this can still be to capture any HTML the user requires.